market/demos/email-encryption/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+	"id": "email-encryption",
+	"name": "Email encryption",
+	"description": "Email encryption is a demo to send an encrypted email to a specific email address.",
```

Comment:
this is not just a demo?

Reply:
yes it is a demo to send an encrypted email.

Reply:
but you also provide a util project which others can use as maven dependency. so its more than a demo.